index.js

```diff
@@ -228,9 +228,11 @@ module.exports = {
     "prefer-const": 2,
     "prefer-reflect": 0,
     "prefer-spread": 0,
-    "prefer-template": 0,
+    "prefer-template": 2,
```

Comment:
Prefer string templates instead of concatenation, makes the following an error:

```javascript
var str = "Hello, " + name + "!";
```

see https://eslint.org/docs/rules/prefer-template for more info